src/Serilog/Configuration/LoggerDestructuringConfiguration.cs

```diff
@@ -161,13 +166,28 @@ public LoggerConfiguration ToMaximumDepth(int maximumDestructuringDepth)
         /// </summary>
         /// <param name="maximumStringLength">The maximum string length.</param>
         /// <returns>Configuration object allowing method chaining.</returns>
-        /// <exception cref="ArgumentOutOfRangeException">When passed length is less or equal to 2</exception>
+        /// <exception cref="ArgumentOutOfRangeException">When passed length is less than 2</exception>
         public LoggerConfiguration ToMaximumStringLength(int maximumStringLength)
         {
             if (maximumStringLength < 2) throw new ArgumentOutOfRangeException(nameof(maximumStringLength), maximumStringLength, "Maximum string length must be at least two.");
             _setMaximumStringLength(maximumStringLength);
             return _loggerConfiguration;
         }
+
+        /// <summary>
+        /// When destructuring objects, collections be restricted to specified length
+        /// thus avoiding bloating payload. Limit is applied to each collection separately, 
+        /// sum of length of collection can exceed limit.
+        /// Applies limit to all <see cref="IEnumerable"/> including dictionaries.
+        /// </summary>
+        /// <returns>Configuration object allowing method chaining.</returns>
+        /// <exception cref="ArgumentOutOfRangeException">When passed length is less than 1</exception>
+        public LoggerConfiguration ToMaximumCollectionLength(int maximumCollectionLength)
```

Comment:
This should possibly be `ToMaximumCollectionSize` or `ToMaximumCollectionCount` since only arrays expose it as `Length` in .NET.